app/controllers/publish_interface/providers_controller.rb

```diff
@@ -30,5 +58,16 @@ def redirect_to_contact_page_with_ukprn_error
 
       redirect_to contact_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)
     end
+
+    def provider_params
+      params
+        .fetch(:publish_interface_about_your_organisation_form, {})
+        .permit(
```

Comment:
I think the form object could declare all these fields in a constant eg `FIELDS` then we could just do a `permit(*PublishInterface::AboutYourOrganisationForm::FIELDS)` to permit whatever is declared from the form? Also means you could do the same with the `attr_accessor` in the form to use the same const